.github/workflows/test-npm.yml

```
@@ -55,6 +55,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     timeout-minutes: 20
     if: ${{ github.repository == 'WordPress/wordpress-develop' || github.event_name == 'pull_request' }}
+    continue-on-error: ${{ matrix.os == 'windows-latest' }}
```

Hmm, wondering if the expression is not returning `true` in the expected type. Let's try this.

```suggestion
    continue-on-error: ${{ matrix.os == 'windows-latest' && true }}
```